### PR TITLE
update recipes for EESSI client containers for 2020.10 pilot

### PR DESCRIPTION
**Pull request by boegel**
container images are already built with the script included here, see https://hub.docker.com/r/eessi/client-pilot/tags

### CODE REVIEW

File: containers/Dockerfile.EESSI-client-pilot-centos7-aarch64 [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM docker.io/arm64v8/centos:7
 RUN yum install -y http://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm \
  && yum install -y cvmfs cvmfs-fuse3 --nogpgcheck \
  && yum install -y cvmfs-config-default sudo vim openssh-clients \
- && yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.1/cvmfs-config-eessi-0.2.1-1.noarch.rpm
+ && yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.2/cvmfs-config-eessi-0.2.2-1.noarch.rpm
```
Comment (boegel) [inline containers/Dockerfile.EESSI-client-pilot-centos7-aarch64:6]: needs another bump (0.2.3?), see #47

**Review by bedroge — APPROVED** [review]
We do still need to update the docs at https://eessi.github.io/docs/pilot/#accessing-the-eessi-pilot-repository-through-singularity (w.r.t. the architecture / URI for the container)

**Comment by boegel** [comment]
> We do still need to update the docs at https://eessi.github.io/docs/pilot/#accessing-the-eessi-pilot-repository-through-singularity (w.r.t. the architecture / URI for the container)

Yes, but let's do that when we actually have stuff installed in `2020.10`?

**Comment by bedroge** [comment]
I also tested it on a RPi, and it works like a charm.

**Pull request merged**